README.md

```diff
@@ -185,13 +88,56 @@ N.B: the amounts listed here are the maximum we can pay for these categories of
 
+## Examples of good bugs
 * https://fin1te.net/articles/uber-turning-self-xss-into-good-xss/
 * http://blog.portswigger.net/2016/04/adapting-angularjs-payloads-to-exploit.html
 * http://blog.orange.tw/2016/04/bug-bounty-uber-ubercom-remote-code_7.html
 * https://labs.integrity.pt/articles/uber-hacking-how-we-found-out-who-you-are-where-you-are-and-where-you-went/
 
------
-*Psst, this is a [Github repo! Wondering if there was a recent rule change? Click here!](https://github.com/uber/Bug-Bounty-Page)*
+# Frequently Asked Questions
+
+## Can I blog about my bug?
+Yes, but we ask that you wait until the issue is both fixed and paid out before you publish the blog post. We also prefer that you request disclosure through HackerOne so that readers of your blog post can get the full background on the issue.
+
+## What is your policy on chaining bugs and privilege escalation?
+Chaining of bugs is not frowned upon in any way, we love to see clever exploit chains! However, if you have managed to compromise an Uber-owned server we do not allow for escalations such as port scanning internal networks, privilege escalation attempts, attempting to pivot to other systems, etc. If you get access to an Uber server please report it us and we will reward you with an appropriate bounty taking into full consideration the severity of what could be done. Chaining a CSRF vulnerability with a self XSS? Nice! Using AWS access key to dump user info? Not cool.
+
+## Do you provide test accounts?
+As of this time we do not have a good system for creating test accounts for our bug bounty submitters. Please create an account as you would normally and perform testing with that account or accounts.  Whenever possible only test against yourself, never other users. If there is ever a situation where you cannot test a bug while adhering to this please let us know and we will help figure out an appropriate solution.
+
+## What is a user UUID and when do you reward for enumeration of it?
+We reward for issues which allow you to enumerate bulk user UUID's via a phone number or email address. For example, if there's an endpoint that allows you to submit an email and the response contains that user's UUID - this is a vulnerability we would reward for. To verify this you should compare the UUID you're seeing with your own email. To get your personal account UUID visit `help.uber.com` after authenticating and copy and paste the following JavaScript in your browser's developer console:
+```js
+alert(JSON.parse($("#web-support-data-script").text).user.uuid);
```

Comment:
our sample JS below uses console.log, but here we use alert. Let's be consistent